### PR TITLE
[WIP]Show what other users are playing on the server

### LINKED ISSUE #769: Show What Other Users Are Playing
It is very nice to see what other users are playing on the server.  Perhaps as a list on the left pane?

### PR DESCRIPTION
**Pull request by Rishabh-malhotraa**
closes #769 

shows what other users are playing in the activity menu.

Screenshot from 2021-04-03 01-39-53

### CODE REVIEW

**Comment by Rishabh-malhotraa** [comment]
This is the way I have currently made it:

Screenshot from 2021-04-03 00-32-15

--- 

### **In the following  album information is also being rendered but i feel it looks a bit messy**

Screenshot from 2021-04-03 00-30-01

**Do let me know whether I should change it to the second image or do something different altogether.**

File: ui/src/layout/ActivityPanel.js [hunk 1]
```diff
@@ -81,7 +87,27 @@ const ActivityPanel = () => {
           dispatch(scanStatusUpdate(data.scanStatus))
         }
       })
-  }, [dispatch])
+    if (open) {
```
Comment (Rishabh-malhotraa) [inline ui/src/layout/ActivityPanel.js:90]: the `useEffect` runs whenever open changes it state where `open = Boolean(anchorEl)` i.e whenever we either open/close the modal but we don't want to make an API call when we close the modal hence this check condtion.

**Comment by Rishabh-malhotraa** [comment]
@deluan there i'm facing a problem when I call `getNowPlaying`

it returns an array, but the array contains only one element always and that is a person who started playing or changed the song, last

I can't figure out why I am not getting the response which has all the user like that mentioned in the docs

I also check it using postman and still the response is only one user when both users are actually playing a song

Screenshot from 2021-04-03 02-28-15

### Response


``` json
{
    "subsonic-response": {
        "status": "ok",
        "version": "1.16.1",
        "type": "navidrome",
        "serverVersion": "dev",
        "nowPlaying": {
            "entry": [
                {
                    "id": "ea53191a02ad3d1307f5cd957472dc5b",
                    "parent": "ba06422cdb46337920f460e02e8ba2b0",
                    "isDir": false,
                    "title": "Come Around Me",
                    "album": "Changes",
                    "artist": "Justin Bieber",
                    "track": 3,
                    "year": 2020,
                    "genre": "Pop",
                    "coverArt": "ea53191a02ad3d1307f5cd957472dc5b",
                    "size": 8285854,
                    "contentType": "audio/mpeg",
                    "suffix": "mp3",
                    "duration": 200,
                    "bitRate": 320,
                    "path": "Justin Bieber/Changes/Come Around Me.mp3",
                    "discNumber": 1,
                    "created": "2021-03-27T13:37:10.975436179Z",
                    "albumId": "ba06422cdb46337920f460e02e8ba2b0",
                    "artistId": "3e4581d5de89a6b4bc12b47dbc8bbf47",
                    "type": "music",
                    "isVideo": false,
                    "username": "admin",
                    "minutesAgo": 4,
                    "playerId": 1,
                    "playerName": "NavidromeUI"
                }
            ]
        }
    }
}
```

**Comment by deluan** [comment]
Hummm.... I took a look in the server code, and it seems that you bumped into a bug. This endpoint was created in 2016 and I never looked into it again after a lot of refactorings, and it seems to not be working properly.... To complete your code, try to mock the response with more info.

In terms of design, maybe add a header, "Now playing", then you don't need to add "current playing" on every line. I think it is important to show Song and  Artist and the time (`minutesAgo`). The Album name is not necessary. We should also truncate long names with ellipsis (`...`).

**Comment by Rishabh-malhotraa** [comment]
deleted the fork so the PR has become orphan, opening a new PR with new changes

**Pull request closed without merging**